src/test/java/seedu/rc4hdb/logic/commands/modelcommands/EditCommandTest.java

```diff
@@ -137,20 +137,20 @@ public void execute_invalidPersonIndexFilteredList_failure() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
         Index outOfBoundIndex = INDEX_SECOND_PERSON;
         // ensures that outOfBoundIndex is still in bounds of address book list
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getPersonList().size());
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getResidentBook().getPersonList().size());
```

Comment:
The `getPersonList` method here should be `getResidentList`, according to the new `ResidentBook` class.